.github/workflows/ci_cd.yml

```
@@ -1312,6 +1312,22 @@ jobs:
           wait_time: 60 # check every minute
           max_time: 1800 # allow up to 30 minutes for a deployment
 
+      - name: Trigger staging k6 load test
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+          owner: WordPress
+          repo: openverse
+          token: ${{ secrets.ACCESS_TOKEN }}
+          workflow_file_name: k6.yml
+          wait_interval: 60
```

@obulat I'd recommend adding this line initially, to prevent the workflow sending those CI failures if on first merging it isn't working well.

Then, after merging, monitor the new k6 workflow itself and see how it goes. Later on, you can make it a stipulation of staging deployment "success" by setting `propagate_failure` to `true`.

```suggestion
          wait_interval: 60
          propagate_failure: false
```

---

That was exactly the change I was going to add, with a doc comment explaining it!

Thank you for your input. I want to get this PR merged next week 🤞